tasks/interfaces.yml

```diff
@@ -1,22 +1,17 @@
 ---
-- name: debian network configuration
+- name: Configure Debian networking
   template:
-    backup: no
+    src: interfaces.j2
```

Comment:
check ansible default mäßig in interfaces? wenn nicht sollte der pfad angepasst werden